src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/TripleColonParser.cs

```diff
@@ -40,7 +40,7 @@ public override BlockState TryOpen(BlockProcessor processor)
             var extensionName = "triple-colon";
             Action<string> logError = (string message) => _context.LogError(
                 $"invalid-{extensionName}",
-                $"Invalid {extensionName} on line {processor.LineIndex}. \"{slice.Text}\" is invalid. {message}",
+                $"Invalid {extensionName} on line {processor.LineIndex}. {message}",
```

Comment:
There is a dedicated line number column to show where the error is on build report, and we'll respect that line when showing the warning in vscode. Suggest to remove the line info from error message.